code/game/objects/items/weapons/implants/implant_track.dm

```diff
@@ -3,7 +3,7 @@
 	desc = "Track with this."
 	activated = 0
 	origin_tech = "materials=2;magnets=2;programming=2;biotech=2"
-	var/id = 1
+	var/id = 1	//this should be redundent now but I'm leaving it incase its still used somewhere in the spaghetti we call ss13 code
```

Comment:
comment it out and compile, if it complains, fix it, if it doesn't complain, remove the commented line altogether.